### ISSUE TITLE
I want to choose the worker through AuthorizeSession Api

### ISSUE THREAD

**Issue by kcoderk** [hashicorp/boundary#2402]
Could we have the feature like that I can choose the worker by API ,rather than only bind it to the target worker filter, that way,we can choose any worker we want to connect to the target

**Comment by kcoderk** [comment]
> Hi @kcoderk, Thank you for the issue. Could you please provide additional information about your use case? Why would it be beneficial for an end user, as opposed to the admin, to be able to select a worker or the set of workers with which they connect to a target?

Yes,you r right,admin can  select a worker they want, for example,I am In SD-WAN,I just want to choose a nearest worker to connect to my target ,I don't care where the target is,I am sure I can access it from this worker,That's what I want,thank you

**Comment by PPacent** [comment]
@kcoderk we will keep this request open to gauge interest from the community based on the upvotes it receives. Please note though that in the meantime, a viable workaround would be creating separate targets with geography-specific worker filters. In this case your target would be configured with a worker filter that selects workers in a specific geography. You can learn more about worker tags and filters here: https://www.boundaryproject.io/docs/concepts/filtering/worker-tags

**Comment by kcoderk** [comment]
> 

Ya,I have experience about worker-tags,the feature I think is not enough to solve the communication issue in cross-region ,but thank you for reply
